charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -37,31 +37,23 @@ spec:
           {{- end }}
           command: ["/bin/bash"]
           {{- if eq .Values.backend.settings "prod" }}
-          args: ["-c", "python manage.py migrate && python3 manage.py collectstatic --noinput && uwsgi --http :8000 --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.{{ .Values.backend.settings }} "]
+          args: ["-c", "python manage.py migrate && python manage.py collectstatic --noinput && uwsgi --http :{{ .Values.backend.service.port }} --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.prod "]
```

Comment:
Why did you modified the settings used here? Is it really always `prod` or is it a typo?

Reply:
We have the condition `{{- if eq .Values.backend.settings "prod" }}` so it will always be prod :)

Reply:
Good point xD